src/main/java/org/skriptlang/skriptworldguard/elements/conditions/CondCanBuildInRegions.java

```diff
@@ -66,7 +65,7 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, @NotNull Kleenean
 	@Override
 	public boolean check(@NotNull Event event) {
 		if (locations != null) {
-			Location[] locations = this.locations.getAll(event);
+			Location[] locations = this.locations.getAll(event); // get all to avoid double-eval + permit or lists.

```

Comment:
hmm do we want to allow or lists? e.g. `player can build in {_region1} or {_region2}` vs `player can build in {_region1} and {_region2}`

Reply:
i'm not sure what i meant by this